### PR TITLE
Add How To Acknowledge Contributors file

### PR DESCRIPTION
**Pull request by SaranjeetKaur**

### CODE REVIEW

File: HOWTO-acknowledge-contributors.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+To recognise the contributions made on this repository we use the [All Contributors](https://allcontributors.org/) Bot. 
+Our aim is to recognise all contributors including those that don't push code. 
+Some (but not limited to) possible contributions which the bot supports include:
```
Comment (hturner) [inline HOWTO-acknowledge-contributors.md:3]:
```suggestion
Types of contribution supported by the bot include:
```

File: HOWTO-acknowledge-contributors.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+- Creating logo/iconography/visual design/etc. for this project
+- Organising events related to this project
+- Providing financial support for this project 
+- Support the infrastructure required for this project
+- Help with the maintenance of this repository
+- Promoting this repository and project
+- Answering Questions in Issues, Slack, etc.
+- Translating content of this repository
+- Conducting literature review for the work on this project
+
```
Comment (hturner) [inline HOWTO-acknowledge-contributors.md:21]: Can you add a link at the end of where to look for a full list (although this is already very comprehensive!).

File: HOWTO-acknowledge-contributors.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+# How to add Contributor(s)?
+
+1. To add a Contributor to this repository, comment on the relevant Issue or Pull Request, asking `@all-contributors` to add a Contributor:
+
+   ```sh
+   @all-contributors please add @<github-username> for <contribution>
+   ```
+
+2. After this the bot will The bot create a Pull Request to add the Contributor. Further it reply with the pull request details.
```
Comment (hturner) [inline HOWTO-acknowledge-contributors.md:30]:
```suggestion
2. After this the bot will create a Pull Request to add the Contributor and reply to your comment with the pull request details. 
```

**Comment by SaranjeetKaur** [comment]
Merging this!

**Pull request merged**